sos/report/plugins/infinidat.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2024 Alejandro Santoyo <alejandro.santoyo@canonical.com>
+#
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+
+import platform
+from sos.report.plugins import Plugin, IndependentPlugin
+    
+    
+class InfinidatStoragePlugin(Plugin, IndependentPlugin):
+    
+    short_desc = 'Infinidat Storage plugin'
+    plugin_name = 'infinidat'
+    profiles = ('storage')
+    packages = ('host-power-tools',)
+
+    def setup(self):
+        # Get infinidat logs
+        self.add_copy_spec("/var/log/infinihost*.log")
+        
+        # Get info from the infinidat boxes, etc.  
+        self.add_cmd_output([
+            "infinihost volume list",
+            "infinihost connectivity list",
+            "infinihost system list",
+            "infinihost pool list",
+            "infinihost snapshot list",
```

Comment:
Is there any inspection we would want to do on the output of these list commands? E.G. if `infinihost volume list` returns one or more volumes, are there any further commands we should run for each of the volumes reported?

Reply:
I've checked and there are no available/useful commands to follow up on the above (nothing like for example `infinihost volume show <volume>`). Hence, I believe these should suffice.